README.md

```diff
@@ -762,6 +763,25 @@ variables:
   ENV: "KEY1=value1,KEY2=value2"
 ```
 
+## Google Dataflow
+
+```yaml
+include:
+  - remote: 'https://raw.githubusercontent.com/jobtome-labs/ci-templates/<REF>/dataflow.yml'
+
+stages:
+  - deploy
+
+variables:
+  GOOGLE_KEY: <google json key>
+  TEMPLATE_NAME: my-template
+  GOOGLE_PROJECT: my-project
+  STAGING_LOCATION: gs://bucket
+  TEMP_LOCATION: gs://bucket
```

Comment:
temp = temporary? not template (which is defined below), right?

Reply:
I've changed this variable to be more clear